### ISSUE TITLE
tests: a11y

### ISSUE THREAD

**Issue by DanielRuf** [zurb/foundation-sites#11053]
We may have to use a11y tools for automated tests eg with [ally.js](https://github.com/medialize/ally.js), [pa11y](https://github.com/pa11y/pa11y) or some other.

Any proposals / suggestions @Owlbertz?

**Comment by Owlbertz** [comment]
I believe those tools will only test on application level, but not on framework level as they evaluate full HTML pages (which is necessary as you cannot get all information on a11y if you do not consider HTML, JS and CSS altogether).

This being said, the question for me is what we want to test using these tools. The docs? Seperate (manual) test pages that we created?
Imo we will still need to have separate unittests for JS and SCSS that check for certain things (contrast, attributes, etc).

Another tool would be [axe-core](https://github.com/dequelabs/axe-core).

**Comment by DanielRuf** [comment]
I thought about testing the components and the defaults.
These can be tested as part of a testsuite.

The docs are another part and will be separately tested with the other tools.

We just need automated tests for the framework to solve most a11y issues at the root.
